xds/core/v3/collection_entry.proto

```diff
@@ -20,15 +20,17 @@ option (xds.annotations.v3.file_status).work_in_progress = true;
 // appearing inside a list collection resource. List collection resources are
 // regular Resource messages of type:
 //
+// ```
 // message <T>Collection {
 //   repeated CollectionEntry resources = 1;
 // }
+// ```
 //
 message CollectionEntry {
   // Inlined resource entry.
   message InlineEntry {
-    // Optional name to describe the inlined resource. Resource names must
-    // [a-zA-Z0-9_-\./]+ (TODO(htuch): turn this into a PGV constraint once
+    // Optional name to describe the inlined resource. Resource names must match
+    // `[a-zA-Z0-9_-\./]+` (TODO(htuch): turn this into a PGV constraint once
```

Comment:
use double backticks ` `` ` for literals in RST - the default with a single backtick is to italicize